pyodide-build/pyodide_build/out_of_tree/pypi.py

```diff
@@ -336,7 +336,7 @@ def _resolve_and_build(
     requirements = []
 
     target_env = {
-        "python_version": f'{common.get_make_flag("PYMAJOR")}.{common.get_make_flag("PYMINOR")}',
+        "python_version": f'{common.get_build_flag("PYMAJOR")}.{common.get_build_flag("PYMINOR")}',
```

Comment:
While you're at it why not add a function for this to common?

Reply:
Sounds good.